### PR TITLE
[Feat]: Display error msg when email is not valid in cons submissions

### PR DESCRIPTION
**Pull request by AlicanAkyuz**
### Description 

This PR updates the Contact Information step of the consignment submission flow in order to display an errror message under the email input when user types in an invalid email address. 

#### Video 

https://user-images.githubusercontent.com/42584148/150965202-a119daee-ff98-4dce-a12c-f1da67bfa7d4.mov

### CODE REVIEW

File: src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx [hunk 1]
```diff
@@ -63,6 +63,13 @@ export const ContactInformationForm: React.FC<ContactInformationFormProps> = ({
         onBlur={handleBlur}
       />
 
+      {/* render email error separately to have its design aligned with PhoneNumberInput */}
```
Comment (kiryl-zubarau) [inline src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx:66]: Have you tried to use input's `error` prop? if PhoneNumberInput error and Input error different let's update PhoneNumberInput error.
`error={touched.email && errors.email}`

File: src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx [hunk 2]
```diff
@@ -63,6 +63,13 @@ export const ContactInformationForm: React.FC<ContactInformationFormProps> = ({
...
+      {errors.email && values.email && (
```
Comment (kiryl-zubarau) [inline src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx:67]: I suppose we also want to show the error if the field was touched(after input loses focus)

File: src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx (hunk 1 above)
Comment (AlicanAkyuz) [reply src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx:66]: Yes, I've tried. if we use the error prop and want to have email and phone number input styles to be the same, we would need to:

- update the styles of Select component (for red border) in PhoneNumberInput
- add logic to dynamically update those styles when the Input next to Select is hovered on 

which is an overkill, in my opinion. but I'm fine with your suggestion if you think going through styles updates makes sense.

Comment (AlicanAkyuz) [reply src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx:66]: ok I'll give it a try

Comment (AlicanAkyuz) [reply src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx]: this works! thanks

**Review by kiryl-zubarau — APPROVED** [review]

**Review by iskounen — APPROVED** [review]

**Review by kiryl-zubarau — APPROVED** [review]

**Pull request merged**